packages/nx/src/command-line/graph/graph.ts

```diff
@@ -238,10 +241,7 @@ export async function generateGraph(
     }
   }
 
-  let html = readFileSync(
-    join(__dirname, '../../core/graph/index.html'),
-    'utf-8'
-  );
+  let html = readFileSync(join(__dirname, '../core/graph/index.html'), 'utf-8');
```

Comment:
These paths seem incorrect now. This would resolve to `packages/nx/src/command-line/core`, instead of `packages/nx/src/core` right?